### ISSUE TITLE
[UX] Bigger +/- buttons

### ISSUE THREAD

**Issue by PikPim** [cagnulein/qdomyos-zwift#223]
The current buttons to shift resistance up or down (+ or -) and quite small when the app is used on a phone, especially during a workout session.
It could be nice to have the touch zone for those button to be the entire half of tile they're in. 
This feature could be enabled/disabled via an option...

**Comment by cagnulein** [comment]
Yeah make sense. Did you try to use use the zoom settings? Are you in the
beta channel?
R.

Il dom 28 mar 2021, 22:09 PikPim ***@***.***> ha scritto:

**Comment by PikPim** [comment]
I don't have access to the beta channel no... and I couldn't find the zoom settings... where are they?

**Comment by cagnulein** [comment]
Write me a mail to ***@***.*** and i Will add to the beta if
you want to try it out

Il mar 30 mar 2021, 02:03 PikPim ***@***.***> ha scritto:
